Fix assumption that download URLs contain file extension (#2710)
github/codeql-action: CHANGELOG.md

## [UNRELEASED]

> - Fix an issue downloading the CodeQL Bundle from a GitHub Enterprise Server instance where the CodeQL Bundle has been synced to the instance using the [CodeQL Action sync tool](https://github.com/github/codeql-action-sync-tool) and the Actions runner does not have Zstandard installed. [#2710](https://github.com/github/codeql-action/pull/2710)

Author: ```suggestion
- Fix an issue downloading the CodeQL Bundle from a GitHub Enterprise Server instance which occurred when the CodeQL Bundle had been synced to the instance using the [CodeQL Action sync tool](https://github.com/github/codeql-action-sync-tool) and the Actions runner did not have Zstandard installed. [#2710](https://github.com/github/codeql-action/pull/2710)
```
